test/physics/test_bond_based.jl

```diff
@@ -38,4 +38,87 @@ end
     @test εc ≈ 0.6454972243679028
 
     @test_throws ArgumentError material!(body, horizon=2, rho=1, E=1, nu=0.26, Gc=1)
+
+    material!(body, horizon=2, rho=1, E=1, Gc=1)
+    (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
+    @test δ ≈ 2.0
+    @test rho ≈ 1.0
+    @test E ≈ 1.0
+    @test nu ≈ 0.25
+    @test G ≈ 0.4
+    @test K ≈ 0.6666666666666666
+    @test λ ≈ 0.4
+    @test μ ≈ 0.4
+    @test Gc ≈ 1.0
+    @test εc ≈ 0.6454972243679028
+
+    @test_throws ArgumentError material!(body, horizon=2, rho=1, nu=0.25, Gc=1)
+
+    material!(body, horizon=2, rho=1, G=0.4, Gc=1)
+    (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
+    @test δ ≈ 2.0
+    @test rho ≈ 1.0
+    @test E ≈ 1.0
+    @test nu ≈ 0.25
+    @test G ≈ 0.4
+    @test K ≈ 0.6666666666666666
+    @test λ ≈ 0.4
+    @test μ ≈ 0.4
+    @test Gc ≈ 1.0
+    @test εc ≈ 0.6454972243679028
+
+    material!(body, horizon=2, rho=1, K=2/3, Gc=1)
+    (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
+    @test δ ≈ 2.0
+    @test rho ≈ 1.0
+    @test E ≈ 1.0
+    @test nu ≈ 0.25
+    @test G ≈ 0.4
+    @test K ≈ 0.6666666666666666
+    @test λ ≈ 0.4
+    @test μ ≈ 0.4
+    @test Gc ≈ 1.0
+    @test εc ≈ 0.6454972243679028
+
+    material!(body, horizon=2, rho=1, lambda=0.4, Gc=1)
+    (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
+    @test δ ≈ 2.0
+    @test rho ≈ 1.0
+    @test E ≈ 1.0
+    @test nu ≈ 0.25
+    @test G ≈ 0.4
+    @test K ≈ 0.6666666666666666
+    @test λ ≈ 0.4
+    @test μ ≈ 0.4
+    @test Gc ≈ 1.0
+    @test εc ≈ 0.6454972243679028
+
+    material!(body, horizon=2, rho=1, mu=0.4, Gc=1)
+    (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
+    @test δ ≈ 2.0
+    @test rho ≈ 1.0
+    @test E ≈ 1.0
+    @test nu ≈ 0.25
+    @test G ≈ 0.4
+    @test K ≈ 0.6666666666666666
+    @test λ ≈ 0.4
+    @test μ ≈ 0.4
+    @test Gc ≈ 1.0
+    @test εc ≈ 0.6454972243679028
+
+    @test_throws ArgumentError material!(body, horizon=2, rho=1, E=1, G=0.4, Gc=1)
+
+    material!(body, horizon=2, rho=1, G=0.4, nu=0.25, Gc=1)
+    (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
+    @test δ ≈ 2.0
+    @test rho ≈ 1.0
+    @test E ≈ 1.0
+    @test nu ≈ 0.25
+    @test G ≈ 0.4
+    @test K ≈ 0.6666666666666666
+    @test λ ≈ 0.4
+    @test μ ≈ 0.4
+    @test Gc ≈ 1.0
+    @test εc ≈ 0.6454972243679028
+
```

Comment:
Maybe add a test like this:
```suggestion

    material!(body, horizon=2, rho=1, lambda=0.4, mu=0.4)
     (; δ, rho, E, nu, G, K, λ, μ, Gc, εc) = body.point_params[1]
     @test δ ≈ 2.0
     @test rho ≈ 1.0
     @test E ≈ 1.0
     @test nu ≈ 0.25
     @test G ≈ 0.4
     @test K ≈ 0.6666666666666666
     @test λ ≈ 0.4
     @test μ ≈ 0.4
     @test Gc ≈ 0.0
     @test εc ≈ 0.0
```